hawkbit-ui/src/main/resources/messages.properties

```diff
@@ -549,3 +549,11 @@ controller.polling.title=Polling Configuration
 controller.polling.time=Polling Time
 controller.polling.overduetime=Polling Overdue Time
 
+# entity not exist
+target.not.exists=Target {0} not exists. Maybe the target is deleted
+targets.not.exists=Targets not exists. Maybe the targets are deleted
+
+distributionset.not.exists=Distribution set not exists. Maybe the set is deleted
+distributionsets.not.exists=Distribution sets not exists. Maybe the sets are deleted
```

Comment:
If the warning is related to multiple Distribution sets better write: Distribution sets do not exists. Maybe the sets were deleted.